superset-frontend/src/components/Select/styles.tsx

```diff
@@ -55,6 +55,13 @@ export const StyledSelect = styled(AntdSelect, {
     .select-all {
       border-bottom: 1px solid ${theme.colors.grayscale.light3};
     }
+    .ant-select-item.ant-select-item-group {
```

Comment:
I believe these should be moved to the `ColorSchemeControl`:

```
<StyledSelect
  css={css`
    width: 100%;
    <HERE>
  `}
```

Reply:
👍